docs/user/hosting.rst

```diff
@@ -1,28 +1,71 @@
-Documentation Hosting Features
-==============================
+.. old label
+.. _Documentation Hosting Features:
 
-The main way that users interact with your documentation is via the hosted HTML that we serve.
-We support a number of important features that you would expect for a documentation host.
+Hosting features provided by Read the Docs
```

Comment:
Having RTD in the title is a bit odd? It's already autoappended to each title.

Seems like the original title was better and more explicit, and better for SEO (includes Documentation hosting).

Reply:
There are similar reasons for why I called it "Special hosting features" in the navigation label.

Alternatively, putting just "Hosting features" made it sound too generic for me. But okay there's a bit of inflation in the language of both suggestions.

We can also simply call it "Documentation hosting features" again, it's not too bad :)

Reply:
Reverted to old title for now.